build/build.xml

```diff
@@ -774,6 +777,9 @@
       <fileset dir="shared" includes="lib/**" excludes="lib/fonts/**" />
       <fileset dir="shared" includes="modes/**" />
       <fileset file="shared/revisions.txt" />
+
+
```

Comment:
I have added ***two spaces*** here to ***avoid merge conflicts*** with my other pending Pull Requests which modifies this same file.